front_end/panels/ai_chat/ui/SettingsDialog.ts

```
@@ -502,6 +526,8 @@ export class SettingsDialog {
   static #groqNanoModelSelect: any | null = null;
   static #openrouterMiniModelSelect: any | null = null;
   static #openrouterNanoModelSelect: any | null = null;
+  static #browseroperatorMiniModelSelect: any | null = null;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)